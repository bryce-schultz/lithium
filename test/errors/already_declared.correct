error: already_declared.li:6:4: 'test' is already defined
│ fn test()
│ ~~~~~~~~~
│    ^
