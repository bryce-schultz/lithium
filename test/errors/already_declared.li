fn test()
{
    print("test");
}

fn test()
{
    print("test");
}

test();