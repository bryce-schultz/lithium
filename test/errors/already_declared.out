error: already_declared.li:6:1: 'test' is already defined
│ fn test()
│ ~~~~~~~~~
│    ^
test
